Invalid high surrogate failure in System.Reflection.Emit tests

http://dotnet-ci.cloudapp.net/job/dotnet_corefx/job/master/job/centos7.1_release_prtest/659/consoleText
```
System.Reflection.Emit.Tests  Total: 975, Errors: 0, Failed: 0, Skipped: 0, Time: 1.793s
EXEC : warning : Invalid high surrogate character (0xDF36). A high surrogate character must have a value from range (0xD800 - 0xDBFF). [/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/src/System.Reflection.Emit/tests/System.Reflection.Emit.Tests.csproj]
  Finished running tests. End time=05:41:02. Return value was 1
/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/Tools/tests.targets(217,5): warning MSB3073: The command "/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/bin/tests/AnyOS.AnyCPU.Release/System.Reflection.Emit.Tests/netcoreapp1.0/RunTests.sh /mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/packages/" exited with code 1. [/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/src/System.Reflection.Emit/tests/System.Reflection.Emit.Tests.csproj]
/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/Tools/tests.targets(226,5): error : One or more tests failed while running tests from 'System.Reflection.Emit.Tests' please check log for details! [/mnt/resource/j/workspace/dotnet_corefx/master/centos7.1_release_prtest/src/System.Reflection.Emit/tests/System.Reflection.Emit.Tests.csproj]
```

--- a/src/System.Reflection.Emit/tests/FieldBuilder/FieldBuilderSetConstant.cs
+++ b/src/System.Reflection.Emit/tests/FieldBuilder/FieldBuilderSetConstant.cs
@@ -148,7 +148,7 @@ namespace System.Reflection.Emit.Tests
             FieldBuilder field = s_type.DefineField("StringField", typeof(string), FieldAttributes.Public);
             
             field.SetConstant(null);
-            field.SetConstant(_generator.GetString(false, 1, 30));
+            field.SetConstant("TestString");
         }
 
         [Fact]

--- a/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderName.cs
+++ b/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderName.cs
@@ -12,6 +12,8 @@ namespace System.Reflection.Emit.Tests
         public static IEnumerable<object[]> Names_TestData()
         {
             yield return new object[] { "TestName" };
+            yield return new object[] { "\uD800\uDC00" };
+            yield return new object[] { "привет" };
             yield return new object[] { "class" };
             yield return new object[] { new string('a', short.MaxValue) };
         }
@@ -29,6 +31,8 @@ namespace System.Reflection.Emit.Tests
         public void Name_InvalidString()
         {
             // TODO: move into Names_TestData when #7166 is fixed
+            Name("\uDC00");
+            Name("\uD800");
             Name("1A\0\t\v\r\n\n\uDC81\uDC91");
         }
     }

--- a/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderName.cs
+++ b/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderName.cs
@@ -8,12 +8,10 @@ using Xunit;
 namespace System.Reflection.Emit.Tests
 {
     public class PropertyBuilderTest9
-    {
-        private static readonly RandomDataGenerator s_randomDataGenerator = new RandomDataGenerator();
-        
+    {        
         public static IEnumerable<object[]> Names_TestData()
         {
-            yield return new object[] { new string((char)(s_randomDataGenerator.GetInt32() % ushort.MaxValue + 1), 1) + s_randomDataGenerator.GetString(false, 1, 260) };
+            yield return new object[] { "TestName" };
             yield return new object[] { "class" };
             yield return new object[] { new string('a', short.MaxValue) };
         }

--- a/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderSetConstant.cs
+++ b/src/System.Reflection.Emit/tests/PropertyBuilder/PropertyBuilderSetConstant.cs
@@ -9,8 +9,6 @@ namespace System.Reflection.Emit.Tests
 {
     public class PropertyBuilderTest11
     {
-        private readonly RandomDataGenerator _generator = new RandomDataGenerator();
-
         private enum Colors
         {
             Red = 0,

